lib/utils.js

```diff
@@ -64,6 +65,11 @@ const getBabelOptions = function loadBabelOptions(filename, options = {}) {
 }
 
 const getTsJestConfig = function getTsJestConfig(config) {
+  const isUsingTs = resolveTsConfigSync(process.cwd())
```

Comment:
Should support tsconfig not place at root dir.
If run `npm run jest`, `process.cwd()` always be root dir. I put tsconfig file at `<rootDir>/sub`.

Reply:
Yes, this is a problem. We should support something like

```js
// jest.config.js
module.exports = {  
  globals: {
    'vue-jest': {
      tsconfig: './some/path/tsconfig.json'
    }
  }
}
```

Would you like to make a PR? I can review + release when it's done. If I need to do it, it might take a while, I have a few other projects I need to work on like Test Utils 2.0.0 release etc. I'd really appreciate the help!